apisix/plugins/request-id.lua

```diff
@@ -40,7 +40,9 @@ local schema = {
     properties = {
         header_name = {type = "string", default = "X-Request-Id"},
         include_in_response = {type = "boolean", default = true},
-        algorithm = {type = "string", enum = {"uuid", "snowflake", "nanoid"}, default = "uuid"}
+        algorithm = {type = "string", enum = {"uuid", "snowflake", "nanoid", "random_id"}, default = "uuid"},
+        random_id_length = { type = "integer",  minimum = 6, default = 16 },
```

Comment:
The name "random" is confusing. It doesn't show why this change can satisfy the requirement of opentelemetry in https://github.com/apache/apisix/issues/8720. Is it because the other algorithms are not random?

Reply:
In https://github.com/apache/apisix/issues/8720 issue, the user needs a random id that can satisfy the `[0-9a-f]{32}` regular expression. I think that the character content and id length of other algorithms cannot meet the requirements. In this "random_id" algorithm, users only need to set random_id_strs="abcdef0123456789" and random_id_length=32 to meet the requirements.

Reply:
As I explained, the name "random" is confusing. Most of the other algorithms also contain a part which is generated randomly.

Reply:
Do you have some good suggestions?